src/github/deployment.ts

```diff
@@ -44,7 +44,7 @@ export const processDeployment = async (
 
 	logger.info("processing deployment message!");
 
-	const jiraPayload: JiraDeploymentData | undefined = await transformDeployment(newGitHubClient, webhookPayload, jiraHost, logger);
+	const jiraPayload: JiraDeploymentData | undefined = await transformDeployment(newGitHubClient, webhookPayload, logger);
```

Comment:
/nit existing code so 🤷‍♂️, types will be inferred from transformDeployment.
`const jiraPayload = await transformDeployment(newGitHubClient, webhookPayload, logger);`